pkg/tests/tasks/extensions/yaml/mesh.tmpl.yaml

```diff
@@ -23,6 +23,14 @@ spec:
     accessLogging:
       file:
         name: /dev/stdout
+  {{ if .ApplyWasmPluginsToInboundOnly }}
```

Comment:
This will only works from new build of the operator? if the change is not yet merged we should not merge this until we start to test on the operator version that have this change included because if not we will see failures in interop test runs

Reply:
We don't need new operator. You can set whatever env variable you want in all versions of the operator.